### ISSUE TITLE
Identify geolocation variables more readily through displayType annotation (backend)

### ISSUE THREAD

**Issue by bobular** [VEuPathDB/EdaLoadingIssues#22]
The study variable metadata for UMSP currently has a "category" node in the tree called "Geolocation" EUPATH_0043212, which has eight intentionally hidden child variables:

1. latitude (type='number', shape='continuous')
2. longitude (type='longitude', shape='continuous')
3. geohash_1 (type='string', shape='categorical')
4. geohash_2
5. geohash_3 etc to geohash_6

The client needs to know which entities have these geolocation variables available.  (A decision for now signed off by Sheena and co is that there will be zero or one per entity.) 

The client could look for a node with eight hidden children, comprising six strings, one number and a longitude, but it there would be less guess-work and more flexibility in the future if we could use the `displayType` - maybe as follows:

```
parent node: displayType = 'geolocation'

latitude: displayType = 'latitude'
longitude: displayType = 'longitude'

geohash_1: displayType = 'geohash', precision = 1
geohash_2: displayType = 'geohash', precision = 2
geohash_3: displayType = 'geohash', precision = 3
```

Would this be possible?

**Comment by SheenaTomko** [comment]
Discussed in slack and decided to have Jay automate loading of the indicated displayType values for all owl files that include the indicated terms such as latitude and longitude. 

I created a separate [redmine ticket ](https://redmine.apidb.org/issues/46961) for Jay's part in changing how displayType is loaded. 

Keep this ticket to represent the work that @bobular will do once Jay's part is complete. We are going to wait until b57 to do this work.

**Comment by jaycolin** [comment]
These properties are now set in eda-inc for the 5 studies affected: four SCORE studies - niger, seasonal, s. mansoni cross-sectional, and moz; and UMSP

**Comment by ryanrdoherty** [comment]
EdaSubsettingService (which parses the variable metadata from the DB) is now set up to allow (and transfer to the client) six display types:
```
default (or a null value, which maps to this)
hidden (deprecated)
multifilter
geoaggregator
latitude
longitude
```
@bobular we cannot use the precision column as you describe above because precision is number- and integer-specific, and (if I remember correctly) the geohash vars are strings.  We can talk about how to find them.  I assume you cannot simply use the name of the var since it varies across studies?  Perhaps we can introduce a new variable type called geohash that also supports precision.  I worry that type and displayType are being conflated or doubly used here.

**Comment by danicahelb** [comment]
changes all made in database and frontend allows displayType but doesn't actively use it yet. @bobular how essential is this to make maps usable in b57?

**Comment by bobular** [comment]
Some front end work remains as discussed in https://github.com/VEuPathDB/web-eda/issues/979, but why is this ticket here (#22) still open?

**Comment by SheenaTomko** [comment]
Closing this, since Bob is correct we have another ticket for implementation
